README.md

```diff
@@ -53,7 +41,7 @@ performance, small size and all the features listed below.
     to hook into any part of the diff and patch process.
   * Splendid performance. Snabbdom is among the fastest virtual DOM libraries.
   * Patch function with a function signature equivalent to a reduce/scan
-    function. Allows for easier integration with a FRP library.
+          function. Allows for easier integration with a FRP library.
```

Comment:
remark parses and outputs again to the same file. There are [some options](https://github.com/remarkjs/remark/tree/master/packages/remark-stringify#options), but it ends up reformatting some of the content.